demisto_sdk/commands/common/tests/update_id_set_test.py

```diff
@@ -2871,6 +2871,46 @@ def test_get_fields_by_script_argument(task):
 
         result = get_fields_by_script_argument(task)
         assert "field_name" in result
+    
+    
+    custom_fields_with_explicit_method ={
+        "id": "ID",
+        "scriptarguments": {
+            "customFields": {'simple': '${keys}'}
+        },
+    }
+    @staticmethod
+    def test_get_fields_by_script_argument__explicit_method(custom_fields_with_explicit_method):
+        """
+        Given
+            - A playbook task with custom fields that retrieves its values in an explicit way (using the ${} method instead of "get" method)
+        When
+            -  Searching for dependent incident fields in the task script arguments
+        Then
+            -  The function should return an empty set with no errors, since a the custom field - ${} sould be ignored
+        """
+        result = get_fields_by_script_argument(custom_fields_with_explicit_method)
+        assert result ==set()
```

Comment:
```suggestion
    @staticmethod
    def test_get_fields_by_script_argument__explicit_method():
        """
        Given
            - A playbook task with custom fields that retrieves its values in an explicit way (using the ${} method instead of "get" method)
        When
            -  Searching for dependent incident fields in the task script arguments
        Then
            -  The function should return an empty set with no errors, since a the custom field - ${} sould be ignored
        """
            custom_fields_with_explicit_method ={
        "id": "ID",
        "scriptarguments": {
            "customFields": {'simple': '${keys}'}
        },
    }
        result = get_fields_by_script_argument(custom_fields_with_explicit_method)
        assert result ==set()
```

do the same for the other test

Reply:
Done